### DISCUSSION TITLE
pptx do not work with RAG API and Azure OpenAI

### DISCUSSION THREAD

**Discussion by ervet** [danny-avila/LibreChat#2462] [Help Wanted]
Hello,

I have tested the RAG API successfully with docx and pdf files. I have always errors with pptx files. Enable debugging RAG API produces the following error. 

rag_api-1         | 2024-04-19 07:28:15,741 - root - DEBUG - /embed - {'id': '661942f9deb505637145c740', 'username': 'erik', 'provider': 'openid', 'email': 'erik@example.com', 'iat': 1713511686, 'exp': 1713512586}
rag_api-1         | 2024-04-19 07:28:15,742 - multipart.multipart - DEBUG - Calling on_part_begin with no data
rag_api-1         | 2024-04-19 07:28:15,743 - multipart.multipart - DEBUG - Calling on_header_field with data[54:73]
rag_api-1         | 2024-04-19 07:28:15,743 - multipart.multipart - DEBUG - Calling on_header_value with data[75:100]
rag_api-1         | 2024-04-19 07:28:15,743 - multipart.multipart - DEBUG - Calling on_header_end with no data
rag_api-1         | 2024-04-19 07:28:15,743 - multipart.multipart - DEBUG - Calling on_headers_finished with no data
rag_api-1         | 2024-04-19 07:28:15,743 - multipart.multipart - DEBUG - Calling on_part_data with data[104:140]
rag_api-1         | 2024-04-19 07:28:15,743 - multipart.multipart - DEBUG - Calling on_part_end with no data
rag_api-1         | 2024-04-19 07:28:15,743 - multipart.multipart - DEBUG - Calling on_part_begin with no data
rag_api-1         | 2024-04-19 07:28:15,743 - multipart.multipart - DEBUG - Calling on_header_field with data[196:215]
rag_api-1         | 2024-04-19 07:28:15,743 - multipart.multipart - DEBUG - Calling on_header_value with data[217:285]
rag_api-1         | 2024-04-19 07:28:15,743 - multipart.multipart - DEBUG - Calling on_header_end with no data
rag_api-1         | 2024-04-19 07:28:15,744 - multipart.multipart - DEBUG - Calling on_header_field with data[287:299]
rag_api-1         | 2024-04-19 07:28:15,744 - multipart.multipart - DEBUG - Calling on_header_value with data[301:374]
rag_api-1         | 2024-04-19 07:28:15,744 - multipart.multipart - DEBUG - Calling on_header_end with no data
rag_api-1         | 2024-04-19 07:28:15,744 - multipart.multipart - DEBUG - Calling on_headers_finished with no data
rag_api-1         | 2024-04-19 07:28:15,745 - multipart.multipart - DEBUG - Calling on_part_data with data[0:87424]
rag_api-1         | 2024-04-19 07:28:15,747 - multipart.multipart - DEBUG - Calling on_part_data with data[0:63712]
rag_api-1         | 2024-04-19 07:28:15,748 - multipart.multipart - DEBUG - Calling on_part_data with data[0:57572]
rag_api-1         | 2024-04-19 07:28:15,748 - multipart.multipart - DEBUG - Calling on_part_end with no data
rag_api-1         | 2024-04-19 07:28:15,748 - multipart.multipart - DEBUG - Calling on_end with no data
rag_api-1         | 2024-04-19 07:28:15,751 - root - INFO - Request POST http://rag_api:8000/embed - 400
LibreChat-API     | 2024-04-19T07:28:15.759Z error: Request failed with status code 400
LibreChat-API     | 2024-04-19T07:28:15.759Z error: [/files] Error processing file: Request failed with status code 400

Any hints 
Thx 
Erik

**Comment by djuillard** [comment]
Same error for me : 
error: Error embedding file Request failed with status code 400
error: [/files] Error processing file: Request failed with status code 400

I think this thread could be merged with #3049

**Comment by djuillard** [comment]
Actually, it does not seem to work neither with openai directly. Strange...
